### ISSUE TITLE
Update PATH for CUDA toolkit

### ISSUE THREAD

**Issue by cyrilcros** [iot-salzburg/gpu-jupyter#41]
Hi, I realized I am missing access to `nvcc`. There are some environment variables to export to fully use the toolkit: https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#post-installation-actions
Could you consider adding them to the image, maybe by adding the `export` lines to the bash profile?
Thanks for building those images!

EDIT: the libraries are there but I don’t see nvcc?

**Comment by cyrilcros** [comment]
Just realized I need to use the devel branch and build my own image....

**Comment by LinG955** [comment]
> Just realized I need to use the devel branch and build my own image....

Hello, How did you solve the problem？I built my image as follow:
```Dockerfile
FROM cschranz/gpu-jupyter:v1.4_cuda-11.0_ubuntu-20.04_python-only

ENV GRANT_SUDO yes
ENV JUPYTER_ENABLE_LAB yes

ENV PATH $PATH:/usr/local/cuda/bin 
ENV CUDA_HOME /usr/local/cuda
ENV LD_LIBRARY_PATH /usr/local/cuda/lib64:$LD_LIBRARY_PATH


ENV NUMBAPRO_NVVM /usr/local/cuda/nvvm/lib64/libnvvm.so
ENV NUMBAPRO_LIBDEVICE /usr/local/cuda/nvvm/libdevice/

RUN python3 -m pip install opencv-contrib-python
```
But, it not work.
```shell
(base) jovyan@8382018aecb5:~$ env | grep CUDA
NVIDIA_REQUIRE_CUDA=cuda>=11.0 brand=tesla,driver>=418,driver<419 brand=tesla,driver>=440,driver<441 brand=tesla,driver>=450,driver<451
CUDA_VERSION=11.0.3
CUDA_HOME=/usr/local/cuda
(base) jovyan@8382018aecb5:~$ nvcc -V
bash: nvcc: command not found
```

**Comment by LinG955** [comment]
> > Just realized I need to use the devel branch and build my own image....
> 
> Hello, How did you solve the problem？I built my image as follow:
> 
> ```dockerfile
> FROM cschranz/gpu-jupyter:v1.4_cuda-11.0_ubuntu-20.04_python-only
> 
> ENV GRANT_SUDO yes
> ENV JUPYTER_ENABLE_LAB yes
> 
> ENV PATH $PATH:/usr/local/cuda/bin 
> ENV CUDA_HOME /usr/local/cuda
> ENV LD_LIBRARY_PATH /usr/local/cuda/lib64:$LD_LIBRARY_PATH
> 
> 
> ENV NUMBAPRO_NVVM /usr/local/cuda/nvvm/lib64/libnvvm.so
> ENV NUMBAPRO_LIBDEVICE /usr/local/cuda/nvvm/libdevice/
> 
> RUN python3 -m pip install opencv-contrib-python
> ```
> 
> But, it not work.
> 
> ```shell
> (base) jovyan@8382018aecb5:~$ env | grep CUDA
> NVIDIA_REQUIRE_CUDA=cuda>=11.0 brand=tesla,driver>=418,driver<419 brand=tesla,driver>=440,driver<441 brand=tesla,driver>=450,driver<451
> CUDA_VERSION=11.0.3
> CUDA_HOME=/usr/local/cuda
> (base) jovyan@8382018aecb5:~$ nvcc -V
> bash: nvcc: command not found
> ```

I solved it: [nvcc: command not found #62](https://github.com/iot-salzburg/gpu-jupyter/issues/62#issue-934322673)
